soccer/gameplay/plays/restarts/our_free_kick.py

```diff
@@ -2,16 +2,25 @@
 import behavior
 import skills.move
 import skills.pivot_kick
-import tactics.defense
 import constants
 import robocup
 import main
+import tactics.coordinated_pass
+import evaluation.touchpass_positioning
 
 
 class OurFreeKick(standard_play.StandardPlay):
+
+    tpass = evaluation.touchpass_positioning
```

Comment:
What's the reasoning for the `tpass` variable?  Any advantage over just calling `evaluation.touchpass_positioning` below?